.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 env:
   - PYTHON_VERSION=2.7
 before_install:
+  - redis-server --version
```

Comment:
It wasn't an outdated version, just not configured properly. I think I have another pull request doing the same thing.